tests/integration/reqs/test_account_tx.py

```diff
@@ -13,3 +13,43 @@ async def test_basic_functionality(self, client):
             )
         )
         self.assertTrue(response.is_successful())
+
+    @test_async_and_sync(globals())
+    async def test_validate_response_type_v2(self, client):
+        response = await client.request(
+            AccountTx(account=WALLET.address, api_version=2)
+        )
+
+        # use the below proxies to ensure that the correct API response is returned
+        # API v2 returns tx_json field
+        self.assertTrue("tx_json" in response.result["transactions"][0])
```

Comment:
```suggestion
        self.assertIn("tx_json", response.result["transactions"][0])
```
Ditto for all the other tests here

Reply:
fixed in 394d1fd